java/ecj/src/main/java/com/ibm/wala/cast/java/translator/jdt/JDT2CAstUtils.java

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
It looks like a top-level `java` directory was added erroneously in this PR.  Assuming that's correct could we delete it?